.ci/scripts/setup-vulkan-linux-deps.sh

```diff
@@ -0,0 +1,48 @@
+
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -ex
```

Comment:
BTW is this script used for developers (non CI) as well?

Reply:
nope, just for CI